Makefile

```diff
@@ -127,7 +127,7 @@ docker-push:
 
 # builds and runs the test suite
 test: | build deps
-	$(ENV_SCRIPT) nim test $(NIM_PARAMS) waku.nims
+	# $(ENV_SCRIPT) nim test $(NIM_PARAMS) waku.nims
```

Comment:
This is on purpose, if you remove this I'm not sure waku v1 tests are run. Please unstage